packages/block-editor/src/store/defaults.js

```diff
@@ -240,5 +240,7 @@ export const SETTINGS_DEFAULTS = {
 		},
 	],
 
+	metadata: {},
```

Comment:
@andrewserong Is this even needed?

Reply:
Hrm, I don't think so — this set of default appears to be for the block editor settings in general, rather than the block level, so I don't believe it'd be needed.

Reply:
Based on [this](https://github.com/WordPress/gutenberg/pull/42605#issuecomment-1206136384) maybe it is?

@Mamaduka are you able to advise?

Reply:
I don't think so. It is a block-level setting and not the editor.

Reply:
I've removed this change.